solarwinds_apm/sampler.py

```diff
@@ -316,79 +317,31 @@ def create_xtraceoptions_response_value(
 
         return ";".join(response)
 
-    def create_new_trace_state(
+    def calculate_trace_state(
         self,
         decision: dict,
         parent_span_context: SpanContext,
         xtraceoptions: Optional[XTraceOptions] = None,
     ) -> TraceState:
-        """Creates new TraceState based on trace decision, parent span id,
-        and x-trace-options if provided"""
-        trace_state = TraceState(
-            [
-                (
-                    INTL_SWO_TRACESTATE_KEY,
```

Comment:
The sampler no longer sets `INTL_SWO_TRACESTATE_KEY` (`sw`) in tracestate internally. We rely on the propagators to `inject` it into outgoing requests. In the case of valid remote parents, it's the propagators' `extract` that bring it to `should_sample`.

Reply:
This makes me so happy